### ISSUE TITLE
Display chart types as icons at the top of the config panel & use accordions instead of tabs

### ISSUE THREAD

**Issue by PriPatel** [lightdash/lightdash#9569]
https://www.figma.com/file/rlqMIJ4pm3t8aAsiMd3CZJ/Lightdash-Product-New?type=design&node-id=4-79&mode=design&t=eU5N86sswwOTdcLA-0

- [ ] Show chart types up top as icons
- [ ] Change category types tabs to accordions

**Comment by hamzahc1** [comment]
why is this blocked @joaoviana ?

**Comment by joaoviana** [comment]
@hamzahc1 just waiting on Design review from @PriPatel, which will happen on or after Wednesday 10th April!

**Comment by PriPatel** [comment]
@joaoviana Im blocked on other stuff so had a bit of time to come up with some next steps:


- The gap between each icon button should be larger. The start of the first button icon and the end of the last button icon should line up nicely with the panel's header (use the screenshot attached here for reference)
<img width="309" alt="Screenshot 2024-04-08 at 16 41 09" src="https://github.com/lightdash/lightdash/assets/12776522/1337ce0a-bd50-4496-8ad4-0d7c9295cfdf">

- The outline of the buttons should be the old light-grey (again use the attached screenshot for reference.
- The icon should be the old, original size ( im probably wrong but they just look a lot smaller than before!)
- New one! There should only be one accordion open at a time. when I open one, the other one should close
- New one! Move Reference lines into it's own accordion

**Comment by PriPatel** [comment]
Next steps Raw dump: (WIP)

- It's great that the chart config panel /  are the same width now, but when I hover over the chart config edge, it looks like I am able to resize it. When I try and resize it, im actually resizing the metrics/dims panels and its confusing!

**Comment by PriPatel** [comment]
https://www.notion.so/lightdash/Chart-config-final-1627c4b43194469e96a2efeb9f9f70a5

**Comment by PriPatel** [comment]
this has been implemented in the new sql runner. Will spec additional improvements and for this to be reused elsewhere in another ticket.
